src/components/Display.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/prefer-stateless-function */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+
+class Display extends Component {
+  render() {
+    const { result } = this.props;
+    return (
+      <div>
+        {result}
+      </div>
+    );
+  }
+}
+
+Display.propTypes = { result: PropTypes.string.isRequired };
+
```

Comment:
lets also add Default props of `'0'` fir the result [see requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)